x-pack/legacy/plugins/apm/cypress/cypress/plugins/index.js

```diff
@@ -19,36 +19,10 @@
 
 // eslint-disable-next-line import/no-extraneous-dependencies
 const wp = require('@cypress/webpack-preprocessor');
-const fs = require('fs');
 
 module.exports = on => {
-  // add typescript support
   const options = {
-    webpackOptions: {
-      resolve: {
-        extensions: ['.ts', '.tsx', '.js']
-      },
-      module: {
-        rules: [
-          {
-            test: /\.tsx?$/,
-            loader: 'ts-loader',
-            options: { transpileOnly: true }
-          }
-        ]
-      }
-    }
```

Comment:
Moved to `../webpack.config.js`, please check there the configuration for typescript. I'd love hearing your experience in the configuration part, as that is a copy&paste mix of the different examples. 🙏